demo/project_demo12-HopsworksTikTokRecSys/1_pipeline.py

```diff
@@ -0,0 +1,143 @@
+from feldera import FelderaClient, SQLContext, SQLSchema
+from feldera.formats import JSONFormat, JSONUpdateFormat
+import config
+
+client = FelderaClient("http://localhost:8080")
+sql = SQLContext("mil", client)
+
+sql.register_table("interactions",
+    SQLSchema({
+        "interaction_id": "STRING",
```

Comment:
Do all these ids have to be strings? This can be very expensive.

Reply:
I think video_id, user_id could also be `CHAR(6)`.
I don't exactly remember the format of `interaction_id`.

Reply:
Can you give a pointer to the original code? CHAR(6) still compiles into String in Rust. I'd very much prefer integer ids.